From 413ae617fa9fe39c99bc72f4dc5d22ce001c5c1a Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Mon, 30 Nov 2020 15:39:40 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: modify the aclk_top init
 frequency

For high performance:
modify aclk_top_high 300M to 500M.
modify aclk_top_low 200M to 400M.

Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Change-Id: I5ef723ed66893024e4b52e49bafb09c3b002b305
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index cb0199f92ab1..9969a7110c90 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -581,7 +581,7 @@
 			<100000000>, <1000000000>,
 			<1188000000>, <600000000>,
 			<150000000>, <100000000>,
-			<300000000>, <200000000>,
+			<500000000>, <400000000>,
 			<150000000>, <100000000>,
 			<300000000>, <150000000>,
 			<1200000000>;
-- 
2.35.3

